witness/ethereum/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nikolaik/python-nodejs@sha256:215bb6c4992613a9ec69ad372578991ed184de11b1ef33520491bc898e3fe936
+
+COPY ./contracts /eth_wit/contracts
+COPY ./tests /eth_wit/tests
+COPY Pipfile* /eth_wit/
+
+WORKDIR /eth_wit
+
+# Maybe a way to do this with determinism?
+# Does pip support verifying hashes of installed packages?
+RUN pip install pipenv
+RUN pipenv install --deploy --system
+RUN yarn global add ganache-cli
```

Comment:
Adding global might have the same issues as with the `pip install`. I'm pretty sure you can lock to an specific ganache version by adding a `package.json`, the auto generated `yarn.lock` and running locally `yarn add ganache-cli`

Eventually if it's needed to run, it can be achieved with `npx ganache-cli` or `yarn run ganache-cli`